build.fsx

```diff
@@ -31,23 +31,29 @@ let MoveFileTo (source, destination) =
 let env = Environment.GetEnvironmentVariable
 let connectionString =
     "DefaultEndpointsProtocol=https;AccountName=" + (env "FILES_ACCOUNT_NAME") + ";AccountKey=" + (env "FILES_ACCOUNT_KEY")
+let manifestToolDir = "./ManifestTool/"
 let packagesDir = "./packages/"
 let toolsDir = "./buildtools/"
 let platform = getBuildParamOrDefault "platform" "x86"
 let buildDir  = "./dist/build" + platform + "/"
 let testDir   = "./dist/test" + platform + "/"
-let deployDir = "./deploy" + platform + "/"
+let artifactsDir = "./artifacts/"
+let telemetryDir = artifactsDir + "SBOMManifestTelemetry/"
 let downloadDir  = "./dist/download" + platform + "/"
+let manifestToolDll = Path.GetFullPath (manifestToolDir @@ "Microsoft.ManifestTool.dll")
+let packageName = "Azure.Functions.Cli." + platform
 let sigCheckExe = toolsDir @@ "sigcheck.exe"
+let dotnetExe = "dotnet.exe"
 let nugetUri = Uri ("https://dist.nuget.org/win-x86-commandline/v3.5.0/nuget.exe")
 let version = if isNull appVeyorBuildVersion then "1.0.0.0" else appVeyorBuildVersion
```

Comment:
Could you please rename these variable `appVeyorBuildVersion` and `appVeyorBuildVersion` as well? Thanks.

Reply:
Thank you for adding a comment.